src/eventgrid/azext_eventgrid/commands.py

```diff
@@ -58,6 +138,77 @@ def load_command_table(self, _):
         g.command('delete', 'delete')
         g.custom_command('update', 'cli_domain_update')
 
+    with self.command_group('eventgrid topic private-endpoint-connection', private_endpoint_connections_mgmt_util, client_factory=private_endpoint_connections_factory) as g:
+        g.custom_command('show', 'cli_topic_private_endpoint_connection_get')
```

Comment:
Please use `custom_show_command` for this and below custom commands for show.

Reply:
thanks.. fixed.